lib/keycloak/volumes/secrets.env

```diff
@@ -0,0 +1,3 @@
+KEYCLOAK_USER=kcadmin
```

Comment:
storing secrets in a repo file is not recommended, please see https://docs.docker.com/engine/swarm/secrets/